src/audio/mixer/mixer_hifi3.c

```diff
@@ -43,17 +39,15 @@ static void mix_n_s16(struct comp_dev *dev, struct audio_stream __sparse_cache *
 			in[j] = audio_stream_wrap(sources[j], in[j]);
 			nmax = audio_stream_samples_without_wrap_s16(sources[j], in[j]);
 			n = MIN(n, nmax);
-			inu[j] = AE_LA64_PP(in[j]);
 		}
 		m = n >> 2;
-		left = n & 0x03;
 
 		for (i = 0; i < m; i++) {
 			val1 = AE_ZERO32();
 			val2 = AE_ZERO32();
 			for (j = 0; j < num_sources; j++) {
 				/* load four 16 bit samples */
-				AE_LA16X4_IP(sample, inu[j], in[j]);
+				AE_L16X4_IP(sample, in[j], 8);
```

Comment:
Why do you add 8 at the base address?
Can you, please, add a comment to explain this?

Reply:
Done